src/components/screens/PostsFeed/PostsFeed.js

```diff
@@ -15,31 +15,57 @@ type Posts = {
   userName: string
 }
 
-class PostsFeed extends PureComponent<Props>{
-  render(){
-      const {posts} = this.props;
-    return (
-    <View style={styles.container}>
+class PostsFeed extends PureComponent<Props> {
+
+  state = {
+    searchName: '',
+    toggleSearch: false
+  };
+
+  filtering = (posts, query) => {
+    return posts.filter((post) => {
+      const userName = post.userName.toUpperCase();
+      const queryData = query.toUpperCase();
+      return userName.indexOf(queryData) > -1;
+    });
+  };
+
+  _setToggle = () => this.setState(state => ({
```

Comment:
Don’t return anything that you don’t expect to be returned. Use brackets here
=> {
  this.setState ...
}